.travis.yml

```diff
@@ -174,3 +177,5 @@ jobs:
         tags: true
 install:
 - pip install -r requirements-dev.txt
+- pip install uwsgi
+- pip freeze
```

Comment:
@karcaw can you verify the versions you are using?

Reply:
versions of what?

Reply:
This is the `pip freeze` output from Travis

```
alabaster==0.7.12
amqp==2.4.2
aspy.yaml==1.2.0
astroid==1.6.6
atomicwrites==1.2.1
attrs==18.2.0
Babel==2.6.0
backports.functools-lru-cache==1.5
backports.tempfile==1.0
backports.weakref==1.0.post1
billiard==3.6.0.0
celery==4.3.0
certifi==2018.10.15
cfgv==1.6.0
chardet==3.0.4
cheroot==6.5.5
CherryPy==18.1.1
colorama==0.4.1
commonmark==0.8.1
coverage==4.5.3
docutils==0.14
elasticsearch==7.0.0
entrypoints==0.3
flake8==3.7.7
flake8-polyfill==1.0.2
future==0.17.1
identify==1.4.2
idna==2.8
imagesize==1.1.0
importlib-metadata==0.9
importlib-resources==1.0.2
isort==4.3.17
jaraco.functools==2.0
Jinja2==2.10.1
kombu==4.5.0
lazy-object-proxy==1.3.1
mando==0.6.4
MarkupSafe==1.1.1
mccabe==0.6.1
mock==2.0.0
more-itertools==4.3.0
nodeenv==1.3.3
nose==1.3.7
numpy==1.16.2
pacifica-archiveinterface==0.2.0
pacifica-metadata==0.9.2
pacifica-policy==0.5.2
pacifica-uniqueid==0.3.3
packaging==19.0
pbr==5.0.0
peewee==3.9.5
pep257==0.7.0
pipenv==2018.11.26
pluggy==0.8.0
portend==2.4
pre-commit==1.15.2
psycopg2==2.7.7
py==1.7.0
pycodestyle==2.5.0
pyflakes==2.1.1
Pygments==2.3.1
pylint==1.9.4
PyMySQL==0.9.3
pyparsing==2.4.0
pytest==4.3.1
python-dateutil==2.8.0
pytz==2019.1
PyYAML==5.1
radon==3.0.1
recommonmark==0.5.0
requests==2.21.0
six==1.12.0
snowballstemmer==1.2.1
Sphinx==2.0.1
sphinxcontrib-applehelp==1.0.1
sphinxcontrib-devhelp==1.0.1
sphinxcontrib-htmlhelp==1.0.2
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.2
sphinxcontrib-serializinghtml==1.1.3
tempora==1.14.1
toml==0.10.0
urllib3==1.24.2
uWSGI==2.0.18
vine==1.3.0
virtualenv==16.0.0
virtualenv-clone==0.4.0
wrapt==1.11.1
zc.lockfile==1.4
zipp==0.4.0
```